integration_tests/e2e/createContacts.cy.ts

```diff
@@ -17,30 +19,19 @@ context('Create Contacts', () => {
     cy.task('stubTitlesReferenceData')
     cy.task('stubRelationshipReferenceData')
     cy.task('stubPrisonerById', TestData.prisoner())
+
+    cy.signIn()
   })
 
   it('Can create a contact with only required fields with direct link', () => {
-    cy.signIn()
     cy.visit('/prisoner/A1234BC/contacts/create/start')
     cy.task('stubCreateContact', { id: 132456 })
-    Page.verifyOnPage(EnterNamePage) //
-      .enterLastName('Last')
-      .enterFirstName('First')
-      .clickContinue()
 
-    const selectRelationshipPage = new SelectRelationshipPage('Last, First')
-    selectRelationshipPage.checkOnPage()
-    selectRelationshipPage //
-      .hasSelectedRelationshipHint('')
-      .selectRelationship('MOT')
-      .hasSelectedRelationshipHint("Last, First is the prisoner's mother")
-      .clickContinue()
-
-    const enterDobPage = new EnterContactDateOfBirthPage('Last, First')
-    enterDobPage.checkOnPage()
-    enterDobPage //
-      .selectIsKnown('NO')
-      .clickContinue()
+    nameIsFirstLast()
+    relationshipIsMother()
+    isEmergencyContact('NO')
+    isNextOfKin('YES')
+    dobIsUnknown()
 
```

Comment:
Bit uncertain about these shorthand functions? They don't really show the journey happening between pages, so the user flow is a little obscured by what look like page validation functions only? Could the specific pages stay visible, but using chained functions demonstrate a flow? Just a thought - something like: 
`
    const relationShipPage = new EnterDobPage()
        .checkOnPage()
        .selectIsKnown('NO'
        .clickContinue()
   
   const emergencyContactPage = relationShipPage
      .checkOnPage()
      .verifyListContains('MOT')
      .select('MOT')
      .continue()
`
etc. ..